test/e2e/ginkgo-node-2.log

```diff
@@ -0,0 +1,20 @@
+Running Suite: Libpod Suite
```

Comment:
Can you remove this and the other log file from the PR?

Reply:
Done.